linfa-kernel/src/lib.rs

```diff
@@ -53,14 +65,17 @@ where
             deserialize = "KernelMethod<R::Elem>: Deserialize<'de>"
         ))
     )]
+    /// The inner product that will be used by the kernel
     pub method: KernelMethod<K1::Elem>,
 }
 
+/// Type definition of Kernel using `Array2<Float>`
 pub type Kernel<F> = KernelBase<Array2<F>, CsMat<F>>;
+/// Type definition of Kernel using `ArrayView2<Float>`
 pub type KernelView<'a, F> = KernelBase<ArrayView2<'a, F>, CsMatView<'a, F>>;
```

Comment:
Hmm, I suppose that makes sense, although I'm not sure exactly how to phrase that in the description. Any suggestions?

Reply:
Sorry, I made a mistake while writing my previous comment. I meant to make the distinction between Owned/Borrowed inner matrix, not Dense/Sparse. I would write something around the lines of:
```rust
/// Type definition of Kernel that owns/borrows its inner matrix
```
or maybe `has a view of` instead of `borrows`?